src/App.tsx

```diff
@@ -109,6 +110,14 @@ const App = () => (
             ...baseHeaderStyle,
           }}
         />
+        <Stack.Screen
+          name="DocList"
+          component={DocList}
+          options={({ navigation }) => ({
+            title: "Tecnologia Padrão",
+            ...baseHeaderStyle,
+          })}
```

Comment:
`navigation` não está sendo usado (warning no linter):

```suggestion
          options={{
            title: "Exemplos",
            ...baseHeaderStyle,
          }}
```